xarray/core/computation.py

```diff
@@ -1141,9 +1141,18 @@ def apply_ufunc(
     numba.guvectorize
     dask.array.apply_gufunc
     xarray.map_blocks
+
     :ref:`dask.automatic-parallelization`
         User guide describing :py:func:`apply_ufunc` and :py:func:`map_blocks`.
 
+    :ref:`dask.automatic-parallelization`
+        Example guide describing :py:func:`apply_ufunc` for 1d array.
+
+    `*Tutorial on apply_ufunc* <https://tutorial.xarray.dev/advanced/apply_ufunc/simple_numpy_apply_ufunc.html>`_
+        Advanced Tutorial on applying numpy function using ``apply_ufunc``
+
+
+    :doc:`/examples/apply_ufunc_vectorize_1d.html`.
```

Comment:
Let's jsut link to https://tutorial.xarray.dev/advanced/apply_ufunc/apply_ufunc.html , The `examples` link above will eventually be removed.